serveradmin/powerdns/routers.py

```diff
@@ -0,0 +1,22 @@
+class PowerDNSRouter:
```

Comment:
If such routers are needed more often, I can see a possibility to provide something which would make adding such easier. So, just already a hint to poke me, if it is needed. ;)

Reply:
Nothing planned.